core/io/image.cpp

```diff
@@ -492,8 +492,15 @@ static void _convert(int p_width, int p_height, const uint8_t *p_src, uint8_t *p
 			}
 
 			if constexpr (write_gray) {
-				//TODO: not correct grayscale, should use fixed point version of actual weights
-				wofs[0] = uint8_t((uint16_t(rgba[0]) + uint16_t(rgba[1]) + uint16_t(rgba[2])) / 3);
+				//REC.601
+				//const uint8_t luminance = (19595U * rgba[0] + 38470U * rgba[1] + 7471U * rgba[2] + 32768U) >> 16U;
```

Comment:
Is REC.601 code needed here? We typically delete unused code.

Reply:
Yep this was my only question about this commit, other code in the engine also has both versions, is this something we want for the future or not?  I put that there in case we need it for the future though.

Reply:
I think the only places with both versions are in third party code. We definitely prefer to keep a tidy codebase.